fe/fe-core/src/main/java/com/starrocks/load/streamload/StreamLoadManager.java

```diff
@@ -141,6 +141,10 @@ public StreamLoadTask createLoadTask(Database db, String tableName, String label
 
     public void unprotectedCheckMeta(Database db, String tblName)
             throws UserException {
+        if (tblName == null) {
+            throw new AnalysisException("Table name must be specified when calling /begin/transaction/ first time");
```

Comment:
```suggestion
            throw new AnalysisException("Table name must be specified when calling /begin/transaction");
```

Reply:
In the case of multiple channels, only the first channel call `begin/transaction/` will be checked, and the other channels call `begin transaction` will not go to this logic to check tableName

Reply:
If the second channel call `/begin/transaction`, how does this channel know the table is the same as the first channel?

Reply:
OK. I get it.